core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/io/CamelIOHandler.java

```diff
@@ -73,7 +72,7 @@ public CamelFile loadCamelModel(IResource res, IProgressMonitor monitor) {
 
 		return cf;
 	}
-
+	
```

Comment:
I think that this file can be reverted, it will avoid to have it in the changeset of this already big commmit